.drone.yml

```diff
@@ -12,72 +13,89 @@ steps:
       mount:
         - ./.tox
         - ./node_modules
+        - ./.pip-cache
       volumes:
         - /tmp/cache:/cache
 
   - name: setup
-    image: registry.online.ntnu.no/dotkom/onlineweb4-testbase:3.7
+    image: registry.online.ntnu.no/dotkom/onlineweb4-testbase:3.7-poetry
```

Comment:
should probably just replace the testbase?

Reply:
I replaced testbase now, I do not think that should affect existing CI, since tox was always downloaded separate anyway